src/TLuaInterpreter.cpp

```diff
@@ -11344,18 +10836,8 @@ int TLuaInterpreter::getGridMode(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setGridMode
 int TLuaInterpreter::setGridMode(lua_State* L)
 {
-    if (!lua_isnumber(L, 1)) {
-        lua_pushstring(L, "setGridMode: wrong argument type");
-        return lua_error(L);
-    }
-    int area = lua_tointeger(L, 1);
-
-    if (!lua_isboolean(L, 2)) {
-        lua_pushstring(L, "setGridMode: wrong argument type");
-        return lua_error(L);
-    }
-    bool gridMode = lua_toboolean(L, 2);
-
+    int area = getVerifiedInt(L, __func__, 1, "areaID");
+    bool gridMode = getVerifiedBoolean(L, __func__, 2, "true/false");
```

Comment:
`grid mode` rather than `true/false` - like the other cases it is implicit that `true` means grid mode "on".